### ISSUE TITLE
[Chalkboard] Buttons do not show up with the recommended version of font-awesome

### ISSUE THREAD

**Issue by DavidVentura** [rajgoel/reveal.js-plugins#140]
Hi
With the recommended font awesome 4.5.0 the pen and pen-square buttons are not visible. I upgraded it to 5.15.4 and they showed up

**Comment by Turiok** [comment]
Hi,

I check the problem. It seems the name of icon change with the version 4.5.0.
It's not 
```
customcontrols: {
				controls: [
					{
					  id: 'toggle-overview',
					  title: 'Toggle overview (O)',
					  icon: '<i class="fa fa-th"></i>',
					  action: 'Reveal.toggleOverview();'
					}
					,
					{ icon: '<i class="fa fa-pen-square"></i>',
					  title: 'Toggle chalkboard (B)',
					  action: 'RevealChalkboard.toggleChalkboard();'
					},
					{ icon: '<i class="fa fa-pen"></i>',
					  title: 'Toggle notes canvas (C)',
					  action: 'RevealChalkboard.toggleNotesCanvas();'
					}
			]
```

but 

```
customcontrols: {
				controls: [
					{
					  id: 'toggle-overview',
					  title: 'Toggle overview (O)',
					  icon: '<i class="fa fa-th"></i>',
					  action: 'Reveal.toggleOverview();'
					}
					,
					{ icon: '<i class="fa fa-pencil-square"></i>',
					  title: 'Toggle chalkboard (B)',
					  action: 'RevealChalkboard.toggleChalkboard();'
					},
					{ icon: '<i class="fa fa-pencil"></i>',
					  title: 'Toggle notes canvas (C)',
					  action: 'RevealChalkboard.toggleNotesCanvas();'
					}
			]
```

**Comment by xuke-hat** [comment]
I have the same problem with 5.1.0, I fix it by change the version to 4.5.0. Does the name of icon change again?
